docs/python/testing.md

```diff
@@ -163,6 +163,25 @@ If discovery fails (for example, the test framework isn't installed or you have
 
 ![Discovery failure error messaged displayed in the Test Explorer](images/testing/test-discovery-error.png)
 
+### Django unittests
+
+The Python extension also offers support for running Django unittests! However, if you are writing tests for your Django application, there are a few additional steps needed in order to have your tests discovered.
+
+First, ensure you have correctly set `"python.testing.unittestEnabled": true,` in your `settings.json` file. Follow the steps below to add `MANAGE_PY_PATH` as an environment variable:
+
+- Create a file named `.env` at the root of your project.
+- Add `MANAGE_PY_PATH='path-string-to-manange.py-path'` to the `.env` file, replacing `path-string-to-manange.py-path` with the path to your application's `manage.py` file (Note: you can copy the path by right clicking on the file and selecting **Copy Path**).
+- Add the following to your `settings.json` file `"python.envFile": "${workspaceFolder}/.env”` to incorporate this file in your environment (edit path as necessary if `.env` is not at root).
+
```

Comment:
some users have run into the problem that we populate `python.testing.unittestArgs` if they go through the UI setup flow right now and these args don't work for Django. Could we add somewhere to edit that variable to remove args and add any Django necessary args

Reply:
Moved this reminder up to the steps